mvnw

```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```

Comment:
is this mvnw file needed?

Reply:
No, I think you're right. We don't really need this maven wrapper given that we have maven and we will be using a `start.sh` script to run the application. I'll remove it.